tools/build-debian-package.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#
+# Written by Daniil Baturin, 2018
+# This file is public domain
+
+echo "Cleaning up the build"
+git clean -dfx
+git reset --hard
+
+echo "Preparing the build"
+./bootstrap.sh
+./configure --with-pkg-extra-version=$EXTRA_VERSION
```

Comment:
In the original doc, I did on purpose set the `-MyDebPkgVersion` for the extra version, so anyone just
running/copying it without reading it would have that extension (and we would see on bug reports that
this is a self-built package)
In your version, unless someone reads the script, they would have no idea about this name extension and build official looking package versions instead.

Maybe set `EXTRA_VERSION` to `-MyDebPkgVersion` at beginning of script (and force this extension if the script is not read or the parameter overwritten by a script argument?

Reply:
I agree, it was over the top. I've fixed it in the updated PR.